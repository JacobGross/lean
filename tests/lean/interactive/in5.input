VISIT sorry2.lean
WAIT
INFO 4