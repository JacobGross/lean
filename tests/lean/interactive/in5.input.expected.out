-- BEGINWAIT
-- ENDWAIT
-- BEGININFO
-- TYPE|4|0
A → B → A ∧ B
-- ACK
-- IDENTIFIER|4|0
and.intro
-- ACK
-- TYPE|4|10
A
-- ACK
-- SYMBOL|4|10
sorry
-- ACK
-- TYPE|4|16
B
-- ACK
-- SYMBOL|4|16
sorry
-- ACK
-- ENDINFO
